core/vm/runtime/runtime.go

```diff
@@ -113,6 +113,12 @@ func Execute(code, input []byte, cfg *Config) ([]byte, *state.StateDB, error) {
 		vmenv   = NewEnv(cfg)
 		sender  = vm.AccountRef(cfg.Origin)
 	)
+	cfg.State.AddAddrToAccessList(cfg.Origin)
```

Comment:
Shouldn't this be added to `Create` line 138 too?

Reply:
And `Call` line 168

Reply:
yup, thanks!